docker.sh

```diff
@@ -78,38 +88,78 @@ function runInBackground {
+    echo -e "'run-71-coverage' - is running dev env with php7.1, xdebug and attaching tty";
+    echo -e "'build-7' - is running build ant tasks based on php 7"
+    echo -e "'build-7-coverage' - is running build ant tasks based on php 7 with code coverage";
+    echo -e "'run-7' - is running dev env and attaching tty";
+    echo -e "'run-coverage-7' - is running dev env with php7, xdebug and attaching tty";
+    echo -e "'build-56' - is running build ant tasks based on php 5.6"
+    echo -e "'build-56-coverage' - is running build ant tasks based on php 5.6 with code coverage";
+    echo -e "'run-56' - is running dev env and attaching tty php 5.6";
+    echo -e "'run-56-coverage' - is running dev env with php5.6, xdebug and attaching tty";
 fi
 
 case $TASK_NAME in
     'build-images')
         buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "all"
         ;;
-    'build')
+    'build-56')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56"
+        runBuild "${APP_NAME}:${APP_VERSION}-php56"
+        ;;
+    'build-56-coverage')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56xdebug"
+        runBuild "${APP_NAME}:${APP_VERSION}-php56xdebug"
+        ;;
+    'run-56')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56"
+        runInBackground "${APP_NAME}:${APP_VERSION}-php56"
+        ;;
+    'run-56-coverage')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56xdebug"
+        runInBackground "${APP_NAME}:${APP_VERSION}-php56xdebug"
+        ;;
+    'build' | 'build-71')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php71"
+        runBuild "${APP_NAME}:${APP_VERSION}-php71"
+        ;;
+    'build-coverage' | 'build-71-coverage')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php71xdebug"
+        runBuild "${APP_NAME}:${APP_VERSION}-php71xdebug"
+        ;;
+    'run' | 'run-71')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php71"
+        runInBackground "${APP_NAME}:${APP_VERSION}-php71"
+        ;;
+    'run-coverage' | 'run-71-coverage')
+        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php71xdebug"
+        runInBackground "${APP_NAME}:${APP_VERSION}-php71xdebug"
+        ;;
+    'build-7')
         buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php7"
         runBuild "${APP_NAME}:${APP_VERSION}-php7"
         ;;
-    'build-coverage')
+    'build-7-coverage')
         buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php7xdebug"
         runBuild "${APP_NAME}:${APP_VERSION}-php7xdebug"
         ;;
-    'build-56')
-        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56"
-        runBuild "${APP_NAME}:${APP_VERSION}-php56"
-        ;;
-    'run')
+    'run-7')
         buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php7"
         runInBackground "${APP_NAME}:${APP_VERSION}-php7"
         ;;
-    'run-coverage')
+    'run-coverage-7')
```

Comment:
rename `run-7-coverage`, imho `coverage` should be a suffix for builds with `xdebug`